build.gradle

```diff
@@ -33,7 +33,6 @@ dependencies {
 	compile('com.google.oauth-client:google-oauth-client-jetty:1.23.0')
 	compile('com.google.api-client:google-api-client:1.23.0')
 	compile('org.flywaydb:flyway-core:5.0.5')
-	testCompile group: 'com.github.javafaker', name: 'javafaker', version: '0.12'
```

Comment:
acceptance criteria: if no name is provided a default name will be used (existing solution)

Reply:
unit tests are missing, check [Jira](http://jira.greenfox.academy/browse/OPAL-205) definition of done for details.